http/src/main/java/io/crate/rest/CrateRestMainAction.java

```diff
@@ -100,7 +100,9 @@ public CrateRestMainAction(Settings settings, Environment environment, RestContr
         siteDirectory = environment.libFile().resolve("site");
         Boolean esApiEnabled = ES_API_ENABLED_SETTING.get(settings);
         Logger logger = Loggers.getLogger(getClass().getPackage().getName(), settings);
-        logger.info("Elasticsearch HTTP REST API {}enabled", esApiEnabled ? "" : "not ");
+        if (esApiEnabled) {
+            logger.info("Elasticsearch HTTP REST API enabled");
```

Comment:
i think we should now log on WARN level, to indicate taht it is not always a safe option and we only log if enabled. a possible log message could be: "Unofficial Elasticsearch HTTP REST API is enabled"